tests/forms_tests/tests/test_media.py

```diff
@@ -606,7 +606,7 @@ def test_merge(self):
     def test_merge_warning(self):
         msg = "Detected duplicate Media files in an opposite order: [1, 2], [2, 1]"
         with self.assertWarnsMessage(RuntimeWarning, msg):
-            self.assertEqual(Media.merge([1, 2], [2, 1]), [1, 2])
+            self.assertEqual(Media.merge([1, 2], [2, 1], None), [1, 2])
```

Comment:
This is just testing the existing behaviour of stripping `None` which was previously untested.